### ISSUE TITLE
Minor UI bug

### ISSUE THREAD

**Issue by kapolos** [rancher/rancher#19413]
**What kind of request is this (question/bug/enhancement/feature request):**

UI bug

**Steps to reproduce (least amount of steps as possible):**

I shut down a node without removing it from the cluster or cordoning it. Pod usage shows 800% :)

**Result:**

![image](https://user-images.githubusercontent.com/10794126/55634654-97e53480-57c7-11e9-859f-f6fa68ffb51d.png)

**Other details that may be helpful:**

Rancher 2.2.0, single install

**Comment by loganhz** [comment]
@kapolos Does refresh browser help?

**Comment by kapolos** [comment]
@loganhz It's at 878 now.

![image](https://user-images.githubusercontent.com/10794126/55646921-0dabc900-57e5-11e9-9e40-7cc790ece28c.png)

**Comment by tfiduccia** [comment]
@kapolos - I'm unable to reproduce this.  Can you tell me:
1.  What type of cluster
2.  How many nodes
3.  Configuration of the nodes (how many with etcd, control plane & workers)

**Comment by kapolos** [comment]
@tfiduccia Thank you forl ooking into it! After reading your post, I took another look and it was on 3000+%. So I realized then what's going on. There were thousands of Pods stuck on "Terminating" state because there were some `nodeSelector`s that specifically required the dead host. 

After deleting the stuck pods with `kubectl get pods --all-namespaces | grep Terminating | awk '{print $2 " --namespace=" $1}' | xargs -n 2 -P 40 kubectl delete pod --grace-period=0 --force` the % became normal.

![image](https://user-images.githubusercontent.com/10794126/55980823-f7e34b80-5c9d-11e9-8087-b0761deb1ac5.png)

I guess it's technically not an error (since the pods existed, even if stuck in "Terminating"), so I'll close the issue. Thanks again!

**Comment by vincent99** [comment]
There's still some things wrong here, the ring shouldn't be allowed to go around past the end and the "x of y used" is showing "4.8" of 550 used instead of 4800 (or at least "4.8k")

**Comment by davidnuzik** [comment]
I tried every way I could think of and could not reproduce the issue. I followed what was suggested in comment https://github.com/rancher/rancher/issues/19413#issuecomment-482232355 @kapolos was this a custom cluster? imported cluster? I created many, many pods, and deleted my nodes (forcefully delete via aws console for example) and could not reproduce in v2.2.0.

Thanks I appreciate any help. It's my job to try to replicate the issue first then QA validate it.

**Comment by kapolos** [comment]
Hi @davidnuzik 

Doing this on 2.2.4: 

![image](https://user-images.githubusercontent.com/10794126/59553107-c38b5480-8f98-11e9-9965-c0fe248b8060.png)

Results to this:

![image](https://user-images.githubusercontent.com/10794126/59553115-f1709900-8f98-11e9-906c-8ae3d3254082.png)

**Comment by davidnuzik** [comment]
Thanks I was doing the same thing but with 2.2.0 instead of 2.2.4 I'll try 2.2.4 now.

**Comment by davidnuzik** [comment]
I am still unable to reproduce the issue. @loganhz if you have time later, any suggestions on how to reproduce? I have workloads in terminating state. Same configuration my nodes have labels and I set up node scheduling to require these nodes. I shut them down so they stop reporting and terminate a bunch of workloads yet I never see greater than 100% Pod usage. If I have just barely not quite 100% pod usage before shutting down nodes, I do see us go from say 95% to 100% but never past this on v2.2.4.

I suppose if we cannot reproduce the problem we can just validate that we fixed the issue where it is not showing "K" ? I do no observe any functional issues with the fix.
cc: @sangeethah (can't reproduce, tried several times and spent hours on this)

![Screen Shot 2019-06-17 at 9 42 05 AM](https://user-images.githubusercontent.com/45179589/59622802-75d13200-90e7-11e9-8995-cbfde44f81f1.png)

![Screen Shot 2019-06-17 at 9 42 19 AM](https://user-images.githubusercontent.com/45179589/59622806-78cc2280-90e7-11e9-882b-b4aacb91633e.png)

**Comment by kapolos** [comment]
@davidnuzik I think you have to create more pods than your maximum for the visual problem to appear. In my screenshot above where I launched 1000 pods, the max for my cluster was 700.

**Comment by davidnuzik** [comment]
When you have (as an example) only 330 pods available and create a workload deployment for 1000 pods you cannot exceed the max 330 pods; this should not be possible and I could not use more than the amount of pods available in my various clusters I tested with.

**Comment by kapolos** [comment]
@davidnuzik Well, I don't know about "should" but as you see it definitely let's me do it. It may not be managing to actually having them all in a running state but the pods themselves register as entities.
It's a bare metal cluster if that makes any difference. Perhaps manages services like AWS provides have extra checks than vanilla k8s does.

**Comment by davidnuzik** [comment]
@kapolos that must be it. I can try a custom cluster with RKE and see if that works. But if that doesn't work I'm not sure what else I can do. I can basically close the issue as should be fixed. A review of the PR looks good to me and it also got reviewed by senior UI dev.

If I can't repro with custom RKE cluster on some ec2 instances or something then I will close the issue. Thanks for the help along the way :) I appreciate it.

**Comment by davidnuzik** [comment]
Version: Master (v2.3) (6/19/19)

I tried with RKE and no joy. Yes I think we are probably doing something to prevent allowing creating more pods than cluster (RKE?) allows. I also tried with EKS, AKS, GKE, and no joy here as well. Perhaps if I had brought up via kubeadm or something then would probably see the issue rather than using RKE or other hosted providers.

The issue should be resolved. I was able to validate the secondary issue where we did not correctly show the unit (K) when we have greater than 999 pods. This is now working for sure.
![image](https://user-images.githubusercontent.com/45179589/59804294-c93baf80-92a2-11e9-9a7e-afa7084a0392.png)
